modin/experimental/core/execution/native/implementations/hdk_on_native/calcite_serializer.py

```diff
@@ -189,7 +189,10 @@ def serialize_obj(self, obj):
         res = {}
         for k, v in obj.__dict__.items():
             if k[0] != "_":
-                res[k] = self.serialize_item(v)
+                if k == "op" and isinstance(obj, OpExpr) and v == "//":
+                    res[k] = "/"
```

Comment:
Why serialization in this case is done separately and not in `serialize_item` function?

Reply:
serialize_item receives a string. We can not replace any string "//" with "/". This is only done for a single case - the operation name of OpExpr.

Reply:
Why did this new case appear?

Reply:
Added support for floordiv [here](https://github.com/modin-project/modin/pull/6251/files#diff-532f2950a0930fecba54c8ab7a130cd9baa85d9320e8b46d9035931be52bc987R165) and [here](https://github.com/modin-project/modin/pull/6251/files#diff-532f2950a0930fecba54c8ab7a130cd9baa85d9320e8b46d9035931be52bc987R923)

Reply:
Now I see, however it looks like a workaround to the fact that coded operations have appeared that cannot be processed in `serialize_item` function.

Reply:
Right, it's a workaround.